app/gui2/src/util/ast/abstract.ts

```diff
@@ -885,6 +885,23 @@ export class Function extends Ast {
     this.equals_ = equals
     this.body_ = body
   }
+
+  static new(
+    module: MutableModule,
+    name: string,
+    args: Ast[],
+    exprs: Ast[],
+    id?: AstId | undefined,
+  ): Function {
+    const exprs_ = exprs.map((expr) => ({ expression: { node: expr } }))
+    const body = BodyBlock.new(exprs_, module)
+    body.addTrailingNewline(module)
```

Comment:
Could the trailing newline be added to the `exprs_` map directly, instead of using the `addTrailingNewline` method and creating a whole new block?

Reply:
Done